### ISSUE TITLE
Problem with RM4c (62be)

### ISSUE THREAD

**Issue by baldasan** [mlfunston/node-red-contrib-broadlink#66]
Hi,
hope I can ask this here, in case no sorry.
I've an RM4c and try to use in .node-red on a rapberry pi.
I can scan the device (below the paylod received).
cloud=false should mean that I've properly followed instruction to avoid cloud mode.
When I try to use the learning flow example nothing happen (in node settings I'm not using a catalog folder).

As additional information the device is working properly (learning and send) when I use it with a python library.

Is there anything that I could do wrong?
Thanks for any help.

[{"mac":"24dfa....",
	"ip":"192.168.1.4",
	"typeid":"62be",
	"type":"RM4c","name":"智能遥控\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0002",
	"cloud":false,
	"msgdata":[0,0,0, .....]
}]

**Comment by mlfunston** [comment]
Yes, if cloud = false then it should work.

Are there any other messages in the node-red-log when you try using the nodes?

**Comment by baldasan** [comment]
No, I just get a closing connection message some seconds later (probably 10 sec later).
I've also tryed to attach chrome debugger and I do not see any additional messages.

**Comment by mlfunston** [comment]
Yes, the system cleans up the connection and closes it later but there should be some debug messages in the node-red logs. What is the output if you have node-red-log running when you send the commands?

Also, what is the config and node setup you are using?

**Comment by baldasan** [comment]
I do not see any message in the node-red-log after I send the command other then "[info] [RM:ac] Broadlink: Closing Device Connection" after timeout elaspes.
Even before there is no message related to the RM flow (I've also tryed increasing the logging level from debug to trace in nodered settings.js). From the UI looks like the RM node is never activated.

I'm using the learning example flow. In attached images youcan see the flow I'm using and relevant configurations.
![noderedTest1](https://user-images.githubusercontent.com/7190955/102124653-d8c09800-3e48-11eb-9b81-575a98ee1968.jpg)
![noderedTest2](https://user-images.githubusercontent.com/7190955/102124657-d9f1c500-3e48-11eb-8fb6-8091affea68f.JPG)
![noderedTest3](https://user-images.githubusercontent.com/7190955/102124658-d9f1c500-3e48-11eb-9f4b-9287b7eea539.JPG)

thanks
